### DISCUSSION TITLE
Switch and RadioGroup: cursor pointer & margin -> padding

### DISCUSSION THREAD

**Discussion by Toshinaki** [mantinedev/mantine#1358] [Feature requests]
Maybe it's just me, but for me, it's more intuitive to have cursor as pointer on switches and radio buttons.

Currently I'm setting the default styles for Switch and RadioGroup as:
```
<MantineProvider styles={{
  RadioGroup: {
    radioWrapper: { cursor: "pointer", "& *": { cursor: "pointer" }},
  },
  Switch: { root: { cursor: "pointer", "& *": { cursor: "pointer" } } },
}} />
```
The `"& *": { cursor: "pointer" }` is because the `input` and `label`'s cursor is set to `default` by default.

It would be nice if we could have pointer built-in with Mantine.

As I said, "maybe it's just me", so please ignore this if it's not a common case.

===================

Also I noticed a left margin on radio label, so if I clicked on the margin, no click event would be triggered.
(The switch label have left padding instead of margin though...)
This could be confusing, especially when the all cursor is default.

**Comment by rtivital** [comment]
#1036

**Reply by Toshinaki** [reply]
OK, if you are following the design pattern of Microsoft or Apple mentioned by this [article](https://medium.com/simple-human/buttons-shouldnt-have-a-hand-cursor-b11e99ca374b), then I'll go on with my MantineProvider styles solution. But please check the comments of that article if you're interested.

Anyway, thank you for this great framework.

Besides, how about the **radio label margin**?

**Comment by tlqvst** [comment]
I absolutely agree that radio inputs need a left padding instead of margin.
